store/etcd/v3/etcd.go

```
@@ -423,8 +426,7 @@ func (s *EtcdV3) NewLock(key string, options *store.LockOptions) (lock store.Loc
 
 	go func() {
 		<-renewCh
-		session.Close()
-		return
+		_ = session.Close()
```

Same here, why assign?

---

if you want to ignore an error, it's better to do it explicitly.